src/Microsoft.Azure.WebJobs.Logging/Internal/LegacyHack.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace Microsoft.Azure.WebJobs.Logging
+{
+    // LogReader needs to handle older format.
+    // DO this in one central place so that we can remove this class when deprecate the old format. 
```

Comment:
Let's create a bug to remove this, after the old versions are removed from the feed post GA